roles/openshift_excluder/tasks/disable.yml

```diff
@@ -1,47 +1,38 @@
 ---
-# input variables
-# - excluder_package_state
-# - docker_excluder_package_state
-- include: init.yml
+- when: r_openshift_excluder_verify_upgrade
```

Comment:
This is stylistic, but i like how you put when at the top, instead of the bottom.  It's much clearer to me when the block should be run.